api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -1160,14 +1160,14 @@ public static String getOpenmrsLogLocation() {
 	}
 	
 	/**
-	 * Checks whether the current JVM version is at least Java 6.
+	 * Checks whether the current JVM version is at least Java 8.
 	 * 
-	 * @throws ApplicationContextException if the current JVM version is earlier than Java 6
+	 * @throws ApplicationContextException if the current JVM version is earlier than Java 8
 	 */
 	public static void validateJavaVersion() {
-		// check whether the current JVM version is at least Java 6
-		if (JdkVersion.getJavaVersion().matches("1.(0|1|2|3|4|5).(.*)")) {
-			throw new APIException("OpenMRS requires Java 6, but is running under " + JdkVersion.getJavaVersion());
+		// check whether the current JVM version is at least Java 8
+		if (JdkVersion.getJavaVersion().matches("1.(0|1|2|3|4|5|6|7).(.*)")) {
+			throw new APIException("OpenMRS requires Java 8, but is running under " + JdkVersion.getJavaVersion());
```

Comment:
Does openmrs require only java 8,as per the exception message here?

Reply:
"OpenMRS requires Java 8 or later, but is running under " - My English is not the best. Will this version of the message be better?

Reply:
we could use (OpenMRS requires Java 8 and above,but it is running under)

Reply:
Done. Thank you for the hint.

Reply:
@sacull Can you squash your commits into one as suggested by @dkayiwa  in thee link he provided

Reply:
@sherrif10 did you see this? https://talk.openmrs.org/t/rfc-favour-not-squashing-commits-over-squashing-commits/28482

Reply:
@dkayiwa  sure i went through it,However i may come back to you with any suggestion